src/components/ProfilePicker/ProfilePickerReferenceWidget.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+	<div class="profile-reference">
+		<div class="profile-reference__wrapper">
+			<div class="profile-card__header">
```

Comment:
```suggestion
			<div class="profile-reference__card__header">
```

nit: builds a nice BEM hierarchy